packages/sdks/web-component/src/lib/helpers/helpers.ts

```diff
@@ -109,10 +114,17 @@ export function getAnimationDirection(
   return undefined;
 }
 
-export const getRunIdsFromUrl = () => {
-  const [executionId = '', stepId = ''] = (getFlowUrlParam() || '').split('_');
+export const getRunIdsFromUrl = (flowId: string) => {
+  let [executionId = '', stepId = ''] = (getFlowUrlParam() || '').split('_');
+  const executionFlowId = getFlowIdFromExecId(executionId);
+
+  // if the flow id does not match, this execution id is not for this flow
+  if (executionFlowId && executionFlowId !== flowId) {
```

Comment:
Let's also add if `flowId` exists

Reply:
AFAIR if there is no flow id there is an error and the SDK is not running
Added it anyway